### PR TITLE
Remove collection update on get (REVIEW ONLY DO NOT MERGE)

### PR DESCRIPTION
**Pull request by chadicus**
When the messages collection is large, the update will cause locks.  By using only the resetTimestamp we can remove the need to update the collection on get

### CODE REVIEW

File: src/Queue.php [hunk 1]
```diff
@@ -155,14 +152,7 @@ public function get(array $query, $runningResetDuration, $waitDurationInMillis =
             $pollDurationInMillis = 0;
         }
 
-        //reset stuck messages
-        $this->collection->update(
-            ['running' => true, 'resetTimestamp' => ['$lte' => new \MongoDate()]],
-            ['$set' => ['running' => false]],
-            ['multiple' => true]
```
Comment (gaillard) [inline src/Queue.php]: This reset all needed docs though, the findAndModify will only do one, where is the solution to that here?

Comment (chadicus) [inline src/Queue.php]: I do not understand the question

Comment (gaillard) [inline src/Queue.php]: The first update does the reset for all needed docs on the collection, the findandmodify does only one, and only if a boolean condition for running true were added to the query, which would take an OR and break the index.

Comment (chadicus) [inline src/Queue.php]: There will be no more updating the entire collection. The running flag is removed.

Comment (gaillard) [inline src/Queue.php]: Count will not mean the same thing..

**Pull request closed without merging**